admin/src/pages/Creation.vue

```diff
@@ -125,20 +125,9 @@ export default {
           throw new Error(event)
       }
     },
-
-    // updateRadioSelected(obj) {
-    //  this.radioSelectedMass = obj[0]
-    // },
-
     removeAllBookmark() {
-      alert('remove all bookmarks')
```

Comment:
@ogerly Brauchen wir hier ein Modal zur Bestätigung?

Reply:
I prefer it like this